.github/workflows/build-and-test.yml

```diff
@@ -5,6 +5,7 @@ on:
   push:
     branches:
       - master
+      - 95-improve-ci-rendering-tests # TODO: REMOVE ONCE PR CREATED
```

Comment:
Just a reminder to remove this when appropriate

Reply:
you are right this has been removed